lms/static/coffee/spec/instructor_dashboard/student_admin_spec.coffee

```diff
@@ -0,0 +1,166 @@
+describe 'StudentAdmin', ->
```

Comment:
@andy-armstrong I have to add this line in `lms/static/js_test.yml` under lib_paths to load coffee script source files in my javascript tests.
`- coffee/src/instructor_dashboard`